ubuntu/python-conda/README.md

```diff
@@ -0,0 +1,8 @@
+# Conda Python Container
+
+This image is intended as an alternative to the [python](https://github.com/databricks/containers/tree/master/ubuntu/python) docker image. It provides similar functionality but with the latest conda environment.
```

Comment:
```suggestion
This image is intended as an alternative to the [python](https://github.com/databricks/containers/tree/master/ubuntu/python) docker image. It provides similar functionality but uses conda as the package manager and builds a conda environment instead of a virtualenv for Python notebooks.
```

Reply:
The standard python DCS image is still built with conda, so we can't add this in quite yet. The wording will be updated when the training team updates the default python image to use virtualenv